qiskit/visualization/text.py

```diff
@@ -676,6 +677,17 @@ def __init__(
 
         self.initial_state = initial_state
         self.cregbundle = cregbundle
+
+        if self.cregbundle:
+            for layer in nodes:
+                for node in layer:
+                    if self.cregbundle:
+                        self.cregbundle = check_cregbundle(node, self._circuit)
+                    else:
+                        break
+                if not self.cregbundle:
+                    break
```

Comment:
These nested loops are a bit hard to follow. I think it is  a case of [for/else](https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops)